### PR TITLE
Houdini: set frame range for the created composite ROP

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
Quick bug fix for created composite ROP, set its frame range to the frame range of the playbar.

## Testing notes:
1. Create Composite ROP 
2. Its frame range should be same as playbar

### CODE REVIEW

File: openpype/hosts/houdini/plugins/create/create_composite.py [hunk 1]
```diff
@@ -32,9 +32,17 @@ def create(self, subset_name, instance_data, pre_create_data):
...
+
+        frame_range = hou.playbar.frameRange()
+
         parms = {
             "trange": 1,
-            "copoutput": filepath
+            "copoutput": filepath,
+            # Unlike many other ROP nodes the composite node does not default
+            # to expression of $FSTART and $FEND so we preserve that behavior
+            # but do set the range to the frame range of the playbar
```
Comment (BigRoy) [inline openpype/hosts/houdini/plugins/create/create_composite.py:43]: > ..so we preserve that behavior but do set the range to the frame range of the playbar..

This seems to be a bit contradictory. We preserve it **but** do it differently?

How about:

> To match it more we at least default to the current playbar frame range
> on creation.

---

Or the question maybe is - if the others default to `$FSTART` and `$FEND` - why wouldn't we do that here too?

Comment (MustafaJafar) [reply openpype/hosts/houdini/plugins/create/create_composite.py:43]: tbh, I just stole these lines from here.
https://github.com/ynput/OpenPype/blob/3817ed150d1cd8a1fa890b673a8a046fdaf72f59/openpype/hosts/houdini/plugins/create/create_review.py#L45-L47

Comment (BigRoy) [reply openpype/hosts/houdini/plugins/create/create_composite.py:43]: Seems I should've had more coffee that day. ;)

**Review by BigRoy — APPROVED** [review]
Didn't test but looks good to me. 👍

**Review by moonyuet — APPROVED** [review]
Looks good for me
image
image
but there is an error for loading the sequences from the export from COP2 Network. and this is not related to this PR
image

**Comment by MustafaJafar** [comment]
> but there is an error for loading the sequences from the export from COP2 Network. and this is not related to this PR
image

This is weird, I faced the same error. 
and here how are my published files look! 
image

**Comment by MustafaJafar** [comment]
> > but there is an error for loading the sequences from the export from COP2 Network. and this is not related to this PR
> > image
> 
> This is weird, I faced the same error. and here how are my published files look! image

I found out the bug (by accident), collector `CollectFrames` should be triggered after `CollectRopFrameRange` 
and I also found the bug responsible for the error related to loading images. 

I will make a PR shortly.

**Pull request merged**